tensorflow_addons/layers/multihead_attention.py

```diff
@@ -26,34 +26,26 @@ class MultiHeadAttention(tf.keras.layers.Layer):
     [Attention Is All You Need](https://arxiv.org/abs/1706.03762) which takes
     in the tensors `query`, `key`, and `value`, and returns the dot-product attention
     between them:
+    >>> mha = MultiHeadAttention(head_size=128, num_heads=12)
+    >>> query = tf.random.uniform((3, 5, 4)) # (batch_size, query_elements, query_depth)
+    >>> key = tf.random.uniform((3, 6, 5)) # (batch_size, key_elements, key_depth)
+    >>> value = tf.random.uniform((3, 6, 6)) # (batch_size, key_elements, value_depth)
 
-    ```python
-    mha = MultiHeadAttention(head_size=128, num_heads=12)
-
-    query = tf.random.uniform((32, 20, 200)) # (batch_size, query_elements, query_depth)
-    key = tf.random.uniform((32, 15, 300)) # (batch_size, key_elements, key_depth)
-    value = tf.random.uniform((32, 15, 400)) # (batch_size, key_elements, value_depth)
-
-    attention = mha([query, key, value]) # (batch_size, query_elements, value_depth)
-    ```
+    >>> attention = mha([query, key, value]) # (batch_size, query_elements, value_depth)
 
     If `value` is not given then internally `value = key` will be used:
 
-    ```python
-    mha = MultiHeadAttention(head_size=128, num_heads=12)
-
-    query = tf.random.uniform((32, 20, 200)) # (batch_size, query_elements, query_depth)
-    key = tf.random.uniform((32, 15, 300)) # (batch_size, key_elements, key_depth)
-
-    attention = mha([query, key]) # (batch_size, query_elements, key_depth)
-    ```
+    >>> mha = MultiHeadAttention(head_size=128, num_heads=12)
+    >>> query = tf.random.uniform((3, 5, 5)) # (batch_size, query_elements, query_depth)
+    >>> key = tf.random.uniform((3, 6, 10)) # (batch_size, key_elements, key_depth)
+    >>> attention = mha([query, key]) # (batch_size, query_elements, key_depth)
```

Comment:
ditto.

Reply:
The author included it. should I remove it or let it be like this

Reply:
@nataliyah123 can you follow the comments here?

Reply:
@WindQAQ sorry it's not clear to me what you are asking. do you mean that two sets of code in docstring are similar in the file multihead_attention? I am referring to lines 30 to 41 in my branch

Reply:
I mean can you print `attention.shape` as well like
```
>>> attention.shape
```

Reply:
I haven't tried it. let me try and then I will get back to you,

Reply:
@WindQAQ I am getting the answer as this TensorShape([3, 5, 6]) but like my other pr (metrics). it is failing the build.